libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -3313,7 +3223,7 @@
 				GCC_WARN_UNDECLARED_SELECTOR = YES;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
-				HEADER_SEARCH_PATHS = "$(SRCROOT)/../../external/ocmock/Source";
+				HEADER_SEARCH_PATHS = "";
```

Comment:
Do we need any value at all here?

Reply:
Removing it from the UI creates an empty string (reason 9803 why I love `Xcode`). In order to remove the entry, we need to hand-edit it using a text editor and remove the line.

Reply:
@wmathurin Thats the only value that was present here. Was able to compile everything, so I guess not.